sdk/src/test/java/io/opentelemetry/sdk/trace/RecordEventsSpanImplTest.java

```diff
@@ -51,6 +56,8 @@
 @RunWith(JUnit4.class)
 public class RecordEventsSpanImplTest {
   private static final String SPAN_NAME = "MySpanName";
+  private static final String SPAN_NEW_NAME = "NewName";
+  private static final long NANOS_PER_SECOND = (long) 1e9;
   private final TraceId traceId = TestUtils.generateRandomTraceId();
   private final SpanId spanId = TestUtils.generateRandomSpanId();
   private final SpanId parentSpanId = TestUtils.generateRandomSpanId();
```

Comment:
Maybe a test for null parentSpanId?

Reply:
Added in https://github.com/open-telemetry/opentelemetry-java/commit/b7e9be01903fcd5d912a0899c71e39186203e40e.